dsl/src/nativeMain/kotlin/com/freeletics/flowredux/dsl/util/AtomicInt.kt

```diff
@@ -0,0 +1,21 @@
+package com.freeletics.flowredux.dsl.util
+
+import kotlin.native.concurrent.AtomicInt
+
```

Comment:
Do you think it makes sense to write some unit tests for this class? In particular multithreaded ones?

Reply:
As we only have a wrapper and a typealias here tests are redundant in my opinion. We would only test ` kotlin.native.concurrent.AtomicInt` and `java.util.concurrent.atomic.AtomicInteger`, which should have already been tested thoroughly (, at least I hope so 😉)